Randominator/UI/Components/NavMenu.razor

```diff
@@ -54,6 +54,11 @@
                      <span class="bi bi-hypnotize" aria-hidden="true"></span> Dare
                 </NavLink>
             </div>
+            <div class="nav-item px-3 @this.FunnyGeneratorsClass">
+                <NavLink class="nav-link" href="generators/Book-Title/">
+                    <span class="bi bi-hypnotize" aria-hidden="true"></span> Book Title
```

Comment:
Could we use a different icon for this one?
Here's a list: https://icons.getbootstrap.com/

Reply:
"book" it is o7